backend/BudgetMate/data_ingestion/admin.py

```diff
@@ -1,3 +1,78 @@
 from django.contrib import admin
+from .models import (
+    BankBrand,
+    BankAccount,
+    AccountStatement,
+    StatementLine,
+    Category,
+    SubCategory,
+)
 
-# Register your models here.
+
+@admin.register(BankBrand)
+class BankBrandAdmin(admin.ModelAdmin):
+    list_display = ("name",)
+
+
+@admin.register(BankAccount)
+class BankAccountAdmin(admin.ModelAdmin):
+    model = BankAccount
+    list_display = ("account_number", "bank", "user", "description")
+
+
+@admin.register(AccountStatement)
+class AccountStatementAdmin(admin.ModelAdmin):
+    model = AccountStatement
+    list_display = (
+        "statement_type",
+        "start_date",
+        "end_date",
+        "bank_account_number",
+        "bank_name_acc",
+        "user_account",
+    )
+
+    def bank_account_number(self, obj):
+        return obj.bank_account.account_number
+
+    def bank_name_acc(self, obj):
+        return obj.bank_account.bank.name
+
+    def user_account(self, obj):
+        return obj.bank_account.user
+
+
+@admin.register(StatementLine)
+class StatementLineAdmin(admin.ModelAdmin):
+    model = StatementLine
+    list_display = (
+        "account_statement",
+        "operation_type",
+        "amount",
+        "operation_date",
+        "user_account",
+        "category",
+        "sub_category",
+    )
+
+    def user_account(self, obj):
```

Comment:
If you're ultimately displaying the user, why not naming it "user" ? That would be a bit less confusing, because you're displaying the user, and not an account in this field

Reply:
Done